Tutorial8/app/views/access/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1 class="header">Log In</h1>
+
+<%= link_to '<button class="back">Back</button>'.html_safe, :back %>
+
+<div id="form">
+<%= form_tag access_path do %>
+  <div>
+    <%= label_tag :username %><br/> 
+    <%= text_field_tag :username,  params[:username] , class: 'control' , :required => true, :pattern=> '^([a-zA-Z])+([\\w]{2,})+$', placeholder: "Enter username(at least 3 characters, no space allowed)" %>
```

Comment:
change to text_field_tag =>

<%= text_field_tag "username",nil , class: "control", required: true, pattern: '^([a-zA-Z])+([\\w]{2,})+$', placeholder: 'Enter username(at least 3 characters, no space allowed)' %>